tests/.gitignore

```diff
@@ -1,3 +1,2 @@
+**/out/*
```

Comment:
It's recommended to also add `**/diff/` here, as this is where diff images are put currently.